soccer/NewRefereeModule.cpp

```diff
@@ -167,10 +167,10 @@ void NewRefereeModule::run() {
 
         stage = (Stage)packet->wrapper.stage();
         command = (Command)packet->wrapper.command();
-        sent_time = packet->wrapper.packet_timestamp();
+        // sent_time = packet->wrapper.packet_timestamp();
         stage_time_left = packet->wrapper.stage_time_left();
-        command_counter = packet->wrapper.command_counter();
-        command_timestamp = packet->wrapper.command_timestamp();
```

Comment:
I mean they won't optimize them out since its part of an class. I changed it to keep it in, but its sort of unnecessary right now.